google/cloud/bigtable/polling_policy.h

```diff
@@ -107,40 +107,54 @@ class PollingPolicy {
 template <typename Retry = LimitedTimeRetryPolicy,
           typename Backoff = ExponentialBackoffPolicy>
 class GenericPollingPolicy : public PollingPolicy {
+  static_assert(std::is_convertible<Retry*, RPCRetryPolicy*>::value,
+                "Retry must derive from RPCRetryPolicy");
+  static_assert(std::is_convertible<Backoff*, RPCBackoffPolicy*>::value,
+                "Backoff must derive from RPCBackoffPolicy");
+
  public:
   explicit GenericPollingPolicy(internal::RPCPolicyParameters defaults)
       : rpc_retry_policy_(Retry(defaults)),
-        rpc_backoff_policy_(Backoff(defaults)) {}
+        rpc_backoff_policy_(Backoff(defaults)),
```

Comment:
Consider renaming the old variables to `rpc_retry_prototype_` and `rpc_backoff_prototype_` following:

https://en.wikipedia.org/wiki/Prototype_pattern

Reply:
Done.